TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -1256,16 +1256,26 @@ def addMultipointToSinglepointIntent(
             main.cleanup()
             main.exit()
 
-    def removeIntent( self, intentId ):
-        """
-        Remove intent for specified intent id
-
+    def removeIntent( self, intentId, app = 'org.onosproject.cli',
+        purge = False, sync = False ):
+        """
+        Remove intent for specified application id and intent id
+        Optional args:- 
+        -s or --sync: Waits for the removal before returning
+        -p or --purge: Purge the intent from the store after removal  
+        
         Returns:
             main.False on error and
             cli output otherwise
         """
         try:
-            cmdStr = "remove-intent org.onosproject.cli " + str( intentId )
+            cmdStr = "remove-intent "
+            if purge:
+                cmdStr += " -p"
+            if sync:
+                cmdStr += " -s"
+
+            cmdStr += " " + app + " " + str( intentId )
```

Comment:
I had to merge this quickly as it is needed for new CHO. Another solution for this would be
to check the received argument type inside driver function and generate error if the received type is not
the expected one. This check could applied to all our driver functions. Foolproof solution would to check
if the received argument matches with any of the expected values.

>>> print type(s)
<type 'str'>
>>>